app/pages/home/homePageController.ts

```diff
@@ -57,4 +120,26 @@ export class HomePageController {
   public navToProjects = () => {
     this.$state.go('projects');
   };
+
+  public startGuidedTour = () => {
+    this.tourInProgress = true;
+  };
+
+  public handleTourEnd = () => {
+    this.tourInProgress = false;
+  };
+
+  public showAllCatalogs = () => {
+    var allSelector: any = jQuery(this.$element).find('.services-categories a');
+    allSelector[0].click();
```

Comment:
This might change with #72 as the 'All' category will be selected by default and all category cards shown by default.